mapbox/examples/guides/getting-started/step-2.html

```diff
@@ -3,8 +3,8 @@
   
   <head>
```

Comment:
Not for this PR, but we should change the example to use Maps API v2 and add the layer directly from the TileJSON endpoint.

Reply:
If you want us to remember it, please create a task in CH